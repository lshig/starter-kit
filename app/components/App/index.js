'use strict'
import React, {Component} from 'react'
import H1 from '../H1/index'
export default class App extends Component {
  render () {
    return (
      <section>
        <H1 text='Hello, World!' />
      </section>
    )
  }
}
