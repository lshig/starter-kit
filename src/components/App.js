import React, { Fragment } from 'react';
import Heading from './Heading';
import SEO from './SEO';

export default function App() {
  return (
    <Fragment>
      <SEO />
      <Heading text="Hello, World!" />
    </Fragment>
  );
}
