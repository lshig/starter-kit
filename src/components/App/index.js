'use strict'
import React from 'react'
import H1 from '../H1'
const App = () => (
  <section>
    <H1 text='Hello, World!' />
  </section>
)
export default App
