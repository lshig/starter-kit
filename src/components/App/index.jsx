import React from 'react'
import H1 from '../H1'
export default function App () {
  return (
    <section>
      <H1 text="Hello, World!" />
    </section>
  )
}
