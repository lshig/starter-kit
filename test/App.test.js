import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from '../src/components/App';

describe('App', () => {
  describe('basic', () => {
    const wrapper = shallow(<App />);

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
