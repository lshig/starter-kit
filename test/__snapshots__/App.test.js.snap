// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App basic renders 1`] = `
<section>
  <Heading
    text="Hello, World!"
  />
</section>
`;
